web/src/add_subscribers_pill.ts

```diff
@@ -101,19 +101,22 @@ export function generate_pill_html(item: CombinedPill): string {
 }
 
 export function set_up_handlers_for_add_button_state(
-    pill_widget: CombinedPillContainer,
+    pill_widget: CombinedPillContainer | user_group_pill.UserGroupPillWidget,
     $pill_container: JQuery,
 ): void {
     const $pill_widget_input = $pill_container.find(".input");
-    const $pill_widget_button = $pill_container.parent().find(".add-users-button");
-
+    const $pill_widget_button = $pill_container.parent().hasClass("add-user-group-container")
+        ? $pill_container.parent().parent().parent().find(".add-groups-button")
```

Comment:
Would be better to just add a common class for all the add buttons and use that directly here.